src/altinn-app-frontend/src/features/form/containers/GroupContainerLikert.test.tsx

```diff
@@ -24,6 +24,22 @@ describe('GroupContainer', () => {
       validateTableLayout(defaultMockQuestions);
     });
 
+    it('should render title, description and left column header', () => {
+      render({
+        likertContainerProps: {
+          textResourceBindings: {
+            title: 'Test title',
+            description: 'Test description',
+            leftColumnHeader: 'Test left column header',
+          },
+        },
+      });
+      screen.getByText('Test title');
+      screen.getByRole('table', { name: 'Test title' });
+      screen.getByText('Test description');
+      screen.getByRole('columnheader', { name: 'Test left column header' });
```

Comment:
getByRole will throw an error if it is not found so I don't see the point in adding an extra assertion to something that is allready found in th document? Or is it something I am missing here

Reply:
The advantage is that the purpose of the code will be more explicit, instead of implicit, and we could have avoided this discussion altogether 🙂 

Future potential errors could also be avoided, if f.ex the `getByRole` is changed (or if the implementation is changed to f.ex `queryByRole`, which does not throw when no items are found) to not throw an error. If we dont add the expects, the test will pass even if the elements are no longer found.

Reply:
Did a bit digging on this, and seems like there are several others that prefer your approach 🙂, and there even is a [eslint rule for it](https://github.com/testing-library/eslint-plugin-testing-library/blob/main/docs/rules/prefer-explicit-assert.md). Personally, I would prefer sticking with `expect` though.

Reply:
[Using `expect` is also recommended](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-get-variants-as-assertions) by Kent C. Dodds, the original author of testing-library.